workos/portal.py

```diff
@@ -16,6 +17,47 @@ def request_helper(self):
             self._request_helper = RequestHelper()
         return self._request_helper
 
+    def create_organization(self, organization):
+        """Create an organization
+
+        Args:
+            organization (dict) - An organization object
+                organization[domains] (list) - ist of domains that belong to the organization
+                organization[name] (str) - A unique, descriptive name for the organization
+
+        Returns:
+            dict: Created Organization response from WorkOS.
+        """
+        return self.request_helper.request(
+            ORGANIZATIONS_PATH,
+            method=REQUEST_METHOD_POST,
+            params=organization,
+            token=workos.api_key,
+        )
+
+    def generate_link(self, intent, organization, return_url=None):
+        """Generate a link to grant access to an organization's Admin Portal
+
+        Args:
+            intent (str): The access scope for the generated Admin Portal link. Valid values are: ["sso"]
+            organization (string): The ID of the organization the Admin Portal link will be generated for
+            return_url (str): The URL that the end user will be redirected to upon exiting the generated Admin Portal. If none is provided, the default redirect link set in your WorkOS Dashboard will be used. (Optional)
```

Comment:
Could you move `return_url` into a `Kwargs` section?

Reply:
Probably a discussion for some other time but... it's called return_url but we're using the default reDirect_uri if one is not supplied?

Reply:
Had the same question as I was writing this. @maxchehab?

Reply:
We are deciding to keep this as is. [See slack convo](https://work-os.slack.com/archives/C01B0NHQQ6M/p1600113895015200).